Library/SPIClient.m

```diff
@@ -978,14 +1054,14 @@ - (void)setTestMode:(BOOL)testMode {
 
 - (BOOL)hasSerialNumberChanged:(NSString *)updatedSerialNumber {
     if ([_serialNumber isEqualToString:updatedSerialNumber])
-        return false;
+    return false;
```

Comment:
This would be clearer if the body of the 'if' statement had braces {}. Eg this is how heartbleed happened :)

Reply:
it is fixed